src/ui/screens/Parachains/OverviewScreen.tsx

```diff
@@ -80,16 +80,18 @@ const Overview = React.memo(function Overview({
                 <Divider />
                 <Padder scale={1} />
                 <View style={styles.itemRow}>
-                  <View style={styles.leasePeriodContainer}>
-                    <ProgressChartWidget
-                      title={`Lease Period`}
-                      detail={`\n${progressPercent}%`}
-                      data={[progressPercent]}
-                    />
+                  <View style={[globalStyles.spaceBetweenRowContainer, globalStyles.flex]}>
                     <View>
-                      <StatInfoBlock title="Current lease">{String(currentLeasePeriod)}</StatInfoBlock>
-                      <StatInfoBlock title="Total">{totalPeriod}</StatInfoBlock>
-                      <StatInfoBlock title="Remainder">{periodRemainder}</StatInfoBlock>
+                      <ProgressChartWidget
+                        title={`Lease Period`}
+                        detail={`\n${progressPercent}%`}
+                        data={[progressPercent]}
+                      />
+                    </View>
+                    <View style={styles.statBlock}>
+                      <Caption>Current lease : {String(currentLeasePeriod)}</Caption>
+                      <Caption>Total : {totalPeriod}</Caption>
+                      <Caption>Remainder : {periodRemainder}</Caption>
```

Comment:
Does this fit in a small screen? How does it look?

Reply:
It looks like this on 4'5 inch screen: 
<img src="https://user-images.githubusercontent.com/22363352/162572539-1f5ec242-f855-41d3-9967-d2d99ee9c974.png" width="250" />

We need to reorder the UI in a better way to handle it.

Reply:
You can use the current approach for now (display and take space vertically), but properly separate items (label and value)

Reply:
With the latest update this is how the UI looks: 

<img src="https://user-images.githubusercontent.com/22363352/163786290-665fcb13-18c1-4d57-babe-ab26e017671e.png" width="250" />

Reply:
It does not look aligned, let's have a chat to revisit this